packages/block-library/src/page-list-item/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { pages as icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import initBlock from '../utils/init-block';
+import metadata from './block.json';
+import edit from './edit.js';
+
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	__experimentalLabel: ( { label } ) => label,
+	icon,
+	example: {},
+	edit,
+};
+
+export const init = () => initBlock( { name, metadata, settings } );
```

Comment:
Why have a separate file? Why not just call `init` here?

Reply:
This is a pattern across the `block-library` package. It comes from https://github.com/WordPress/gutenberg/pull/42258 which enabled easier importing of only one block.